README.md

```diff
@@ -73,11 +73,13 @@ The built-in integration with Terraform, allows you to easily provision your
 infrastructure using Terraform and let KubeOne take all the needed information
 from the Terraform state.
 
-### Integration With Cluster-API and Kubermatic machine-controller
+### Integration With Cluster-API, Kubermatic machine-controller, and operating-system-manager
 
 Manage your worker nodes declaratively by utilizing the [Cluster-API][cluster-api]
 and [Kubermatic machine-controller][machine-controller]. Create, remove,
 upgrade, or scale your worker nodes using kubectl.
+[Kubermatic operating-system-manager][operating-system-manager] is responsibile for
```

Comment:
Please also create a PR to add this to docs.

Reply:
Yeah, I'll create it later under https://github.com/kubermatic/operating-system-manager/issues/186